examples/pytorch/graphsage/train_full.py

```diff
@@ -167,5 +83,42 @@ def main(args):
                         help="Aggregator type: mean/gcn/pool/lstm")
     args = parser.parse_args()
```

Comment:
I'm not sure if we should have a large number of configurable arguments. This depends on if the README file includes experiment results with different configurations. Otherwise, users should be able to directly modify the script if necessary.

Reply:
In terms of README file, I don't think we need to cover test results with different configurations. We can drop a pointer in README and it should be easy for them to find out what can be/how to configure in the code.

In fact, we should also simplify reported results in README for all cases. Because: (1) results/performance changing rapidly depends on hardware and self-evolving dgl/other libraries; (2) users don't really care/trust what we reported in README. The impression is more from their on-screen observations and own applications.

To sum up, I think, in README, we only need (1) model name, brief description of references; (2) required pkgs (3) sample command line(s) to run examples (4) just approx. train accuracy results for user verification.